flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/FlinkCatalog.java

```diff
@@ -229,6 +229,20 @@ public void dropDatabase(String name, boolean ignoreIfNotExists, boolean cascade
       throws DatabaseNotExistException, DatabaseNotEmptyException, CatalogException {
     if (asNamespaceCatalog != null) {
       try {
+        if (cascade) {
+          List<ObjectPath> deleteTablePaths = icebergCatalog.listTables(toNamespace(name)).stream()
+                  .map(s -> ObjectPath.fromString(s.toString()))
+                  .collect(Collectors.toList());
+          deleteTablePaths.forEach(
+                  objectPath -> {
+                    try {
+                      dropTable(objectPath, true);
+                    } catch (TableNotExistException ex) {
```

Comment:
If we encounter other exception while dropping the table, should we continue to drop other tables, or should we abort the drop operation?

Reply:
We can ignore it, and drop other tables.

Reply:
Is it possible to get an exception other than `TableNotExistException`?

Reply:
Since I made these changes based on the way Flink implements it, I think we should not change it.
![image](https://user-images.githubusercontent.com/74643176/167244998-a5dab84a-e38f-4919-82d1-9326fe4fe518.png)

Reply:
We can certainly be consistent with it, I have no objection. I just want to confirm our deletion policy. For now, when encounter an exception other than A, it will abort the deletion, right?